.github/workflows/comment-trigger.yml

```
@@ -59,14 +59,24 @@ jobs:
           || contains(github.event.comment.body, '/ci-run-regression')
       )
     steps:
-      - name: Validate issue comment
+      - name: Get PR comment author
         id: comment
         uses: tspascoal/get-user-teams-membership@v2
         with:
           username: ${{ github.actor }}
           team: 'Vector'
           GITHUB_TOKEN: ${{ secrets.GH_PAT_ORG }}
 
+      - name: Validate author membership
+        env:
+          isTeamMember: ${{ steps.comment.outputs.isTeamMember }}
+        run: |
+          if [ "${{ steps.comment.outputs.isTeamMember }}" = "false" ] ; then
+            exit 1
+          else
+            exit 0
+          fi
```

We could simplify this to something like the following
```suggestion
      - name: Validate author membership
        if: steps.comment.outputs.isTeamMember == 'false'
        run: exit 1
```

or use `actions/github-scripts@v6` to provide an error message
```
      - name: Validate author membership
        if: steps.comment.outputs.isTeamMember == 'false'
        uses: actions/github-script@v6
        with:
          script: |
              core.setFailed('author is not a vector team member')
```